deploy/lib/ml_rest.rb

```diff
@@ -73,6 +73,37 @@ def install_properties(path)
       end
     end
 
+    def install_options(path)
+      @logger.info("Loading REST options in #{path}")
+      if (File.exists?(path))
+        Dir.foreach(path) do |item|
+          next if item == '.' or item == '..'
+
+          file = open("#{path}/#{item}", "rb")
+          ext = File.extname(item)
+          basename = File.basename(item, ext)
+
+          headers = {}
+          if (ext == '.xml')
+            headers['Content-Type'] = 'application/xml'
+          elsif (ext == '.json')
+            headers['Content-Type'] = 'application/json'
+          else
+            @logger.error("Unrecognized REST options format: #{item}")
+          end
+
+          contents = file.read
+
+          r = go("http://#{@hostname}:#{@port}/v1/config/query/#{basename}", "put", headers, nil, contents)
```

Comment:
Look at how we are doing https or http in server_config.rb. You should probably do that here.